horace_core/sqw/PixelData/@PixelData/private/logical_to_normal_index_.m

```diff
@@ -0,0 +1,15 @@
+function abs_pix_indices = logical_to_normal_index_(obj, logical_indices)
+%ABS_PIX_INDICES Convert the given logical indices to normal indices
+%
```

Comment:
Not clear why it would be ok to have this array larger than expected as long as the extras aren't logicals. It sounds as if it should be an error to get here in the first place